app/features-json/middleware/jwt_auth.rb

```diff
@@ -18,7 +18,7 @@ def call(env)
         begin
           options = { verify_iss: true, verify_iat: true, algorithm: "HS256", iss: "fastlane.ci" }
           bearer = env.fetch("HTTP_AUTHORIZATION", "").slice(7..-1)
-          payload = JWT.decode(bearer, @encryption_key, true, options)
+          payload = JWT.decode(bearer, @encryption_key, true, options)[0]
```

Comment:
let's use named params

Reply:
What is `[0]`? Let's add a comment here

Reply:
why is this being changed? 🤔  better take `payload,` if there multiple return statements from `.decode`